rpgpy/metadata.py

```diff
@@ -305,6 +305,10 @@
         name='covariance_spectrum_re',
         long_name='Covariance Spectrum Re',
         comment='real part, linear Ze'),
+    'LDRSpec': Meta(
+        name='LDR spectrum',
+        long_name='linear depolarisation ratio Doppler spectra',
+        comment='in dB'),
```

Comment:
You can put the unit directly to `units` attribute.

Reply:
I implemented a test, but the lv0 file in the data folder is DualPol=1, so actually the method shouldn't be applied.